[2024-09-19T23:19:01.624+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_ingestion.raw_ingestion manual__2024-09-19T23:19:00.108554+00:00 [queued]>
[2024-09-19T23:19:01.639+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_ingestion.raw_ingestion manual__2024-09-19T23:19:00.108554+00:00 [queued]>
[2024-09-19T23:19:01.639+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 0
[2024-09-19T23:19:01.661+0000] {taskinstance.py:1382} INFO - Executing <Task(SSHOperator): raw_ingestion> on 2024-09-19 23:19:00.108554+00:00
[2024-09-19T23:19:01.669+0000] {standard_task_runner.py:57} INFO - Started process 1791 to run task
[2024-09-19T23:19:01.671+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'raw_ingestion', 'raw_ingestion', 'manual__2024-09-19T23:19:00.108554+00:00', '--job-id', '121', '--raw', '--subdir', 'DAGS_FOLDER/raw_ingestion.py', '--cfg-path', '/tmp/tmpdj768lg5']
[2024-09-19T23:19:01.672+0000] {standard_task_runner.py:85} INFO - Job 121: Subtask raw_ingestion
[2024-09-19T23:19:01.833+0000] {task_command.py:416} INFO - Running <TaskInstance: raw_ingestion.raw_ingestion manual__2024-09-19T23:19:00.108554+00:00 [running]> on host airflow-worker
[2024-09-19T23:19:02.115+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_ingestion' AIRFLOW_CTX_TASK_ID='raw_ingestion' AIRFLOW_CTX_EXECUTION_DATE='2024-09-19T23:19:00.108554+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-19T23:19:00.108554+00:00'
[2024-09-19T23:19:02.116+0000] {ssh.py:141} INFO - Creating ssh_client
[2024-09-19T23:19:02.116+0000] {ssh.py:114} INFO - ssh_hook is not provided or invalid. Trying ssh_conn_id to create SSHHook.
[2024-09-19T23:19:02.167+0000] {base.py:73} INFO - Using connection ID 'spark_ssh_default' for task execution.
[2024-09-19T23:19:02.168+0000] {ssh.py:300} WARNING - No Host Key Verification. This won't protect against Man-In-The-Middle attacks
[2024-09-19T23:19:02.201+0000] {transport.py:1893} INFO - Connected (version 2.0, client OpenSSH_8.9p1)
[2024-09-19T23:19:02.330+0000] {transport.py:1893} INFO - Authentication (password) successful!
[2024-09-19T23:19:02.331+0000] {ssh.py:478} INFO - Running command: export SPARK_HOME=/usr/local/spark; /opt/conda/bin/spark-submit --master local[*] --executor-memory 2g /opt/shared/scripts/raw_ingestion.py
[2024-09-19T23:19:11.252+0000] {ssh.py:524} INFO - 24/09/19 23:19:11 INFO SparkContext: Running Spark version 3.3.1
[2024-09-19T23:19:11.503+0000] {ssh.py:524} INFO - 24/09/19 23:19:11 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-09-19T23:19:12.047+0000] {ssh.py:524} INFO - 24/09/19 23:19:12 INFO ResourceUtils: ==============================================================
[2024-09-19T23:19:12.052+0000] {ssh.py:524} INFO - 24/09/19 23:19:12 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-09-19T23:19:12.052+0000] {ssh.py:524} INFO - 24/09/19 23:19:12 INFO ResourceUtils: ==============================================================
[2024-09-19T23:19:12.057+0000] {ssh.py:524} INFO - 24/09/19 23:19:12 INFO SparkContext: Submitted application: load
[2024-09-19T23:19:12.851+0000] {ssh.py:524} INFO - 24/09/19 23:19:12 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-09-19T23:19:12.939+0000] {ssh.py:524} INFO - 24/09/19 23:19:12 INFO ResourceProfile: Limiting resource is cpu
[2024-09-19T23:19:12.940+0000] {ssh.py:524} INFO - 24/09/19 23:19:12 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-09-19T23:19:13.551+0000] {ssh.py:524} INFO - 24/09/19 23:19:13 INFO SecurityManager: Changing view acls to: root
[2024-09-19T23:19:13.552+0000] {ssh.py:524} INFO - 24/09/19 23:19:13 INFO SecurityManager: Changing modify acls to: root
[2024-09-19T23:19:13.571+0000] {ssh.py:524} INFO - 24/09/19 23:19:13 INFO SecurityManager: Changing view acls groups to: 
[2024-09-19T23:19:13.575+0000] {ssh.py:524} INFO - 24/09/19 23:19:13 INFO SecurityManager: Changing modify acls groups to: 
[2024-09-19T23:19:13.582+0000] {ssh.py:524} INFO - 24/09/19 23:19:13 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(root); groups with view permissions: Set(); users  with modify permissions: Set(root); groups with modify permissions: Set()
[2024-09-19T23:19:16.361+0000] {ssh.py:524} INFO - 24/09/19 23:19:16 INFO Utils: Successfully started service 'sparkDriver' on port 44691.
[2024-09-19T23:19:16.897+0000] {ssh.py:524} INFO - 24/09/19 23:19:16 INFO SparkEnv: Registering MapOutputTracker
[2024-09-19T23:19:17.699+0000] {ssh.py:524} INFO - 24/09/19 23:19:17 INFO SparkEnv: Registering BlockManagerMaster
[2024-09-19T23:19:17.944+0000] {ssh.py:524} INFO - 24/09/19 23:19:17 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-09-19T23:19:17.952+0000] {ssh.py:524} INFO - 24/09/19 23:19:17 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-09-19T23:19:17.980+0000] {ssh.py:524} INFO - 24/09/19 23:19:17 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-09-19T23:19:18.235+0000] {ssh.py:524} INFO - 24/09/19 23:19:18 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-36b5f9d5-a305-4b21-92dd-14e4d6329203
[2024-09-19T23:19:18.406+0000] {ssh.py:524} INFO - 24/09/19 23:19:18 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-09-19T23:19:19.132+0000] {ssh.py:524} INFO - 24/09/19 23:19:19 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-09-19T23:19:20.587+0000] {ssh.py:524} INFO - 24/09/19 23:19:20 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2024-09-19T23:19:20.627+0000] {ssh.py:524} INFO - 24/09/19 23:19:20 INFO Utils: Successfully started service 'SparkUI' on port 4041.
[2024-09-19T23:19:21.413+0000] {ssh.py:524} INFO - 24/09/19 23:19:21 INFO Executor: Starting executor ID driver on host spark-master
[2024-09-19T23:19:21.460+0000] {ssh.py:524} INFO - 24/09/19 23:19:21 INFO Executor: Starting executor with user classpath (userClassPathFirst = false): ''
[2024-09-19T23:19:21.792+0000] {ssh.py:524} INFO - 24/09/19 23:19:21 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 38937.
[2024-09-19T23:19:21.792+0000] {ssh.py:524} INFO - 24/09/19 23:19:21 INFO NettyBlockTransferService: Server created on spark-master:38937
[2024-09-19T23:19:21.803+0000] {ssh.py:524} INFO - 24/09/19 23:19:21 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-09-19T23:19:21.856+0000] {ssh.py:524} INFO - 24/09/19 23:19:21 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, spark-master, 38937, None)
[2024-09-19T23:19:21.876+0000] {ssh.py:524} INFO - 24/09/19 23:19:21 INFO BlockManagerMasterEndpoint: Registering block manager spark-master:38937 with 434.4 MiB RAM, BlockManagerId(driver, spark-master, 38937, None)
[2024-09-19T23:19:21.887+0000] {ssh.py:524} INFO - 24/09/19 23:19:21 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, spark-master, 38937, None)
[2024-09-19T23:19:21.923+0000] {ssh.py:524} INFO - 24/09/19 23:19:21 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, spark-master, 38937, None)
[2024-09-19T23:19:23.210+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 178, in execute
    result = self.run_ssh_client_command(ssh_client, self.command, context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 166, in run_ssh_client_command
    self.raise_for_status(exit_status, agg_stderr, context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 160, in raise_for_status
    raise AirflowException(f"SSH operator error: exit status = {exit_status}")
airflow.exceptions.AirflowException: SSH operator error: exit status = -1
[2024-09-19T23:19:23.219+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=raw_ingestion, task_id=raw_ingestion, execution_date=20240919T231900, start_date=20240919T231901, end_date=20240919T231923
[2024-09-19T23:19:23.306+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 121 for task raw_ingestion (SSH operator error: exit status = -1; 1791)
[2024-09-19T23:19:23.336+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-09-19T23:19:23.473+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
